_posts/2022-09-15-homebrew-maintainer-projects-update.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Maintainer Projects
+author: SMillerDev
+---
+
+Homebrew’s Project Leadership Committee has green-lit two paid projects by our maintainers this year and since both have hit some milestones recently we’d love to give you, our sponsors and users, an update on their progress.
+
+## API Project
+Since version [3.6.0]({% post_url 2022-09-07-homebrew-3.6.0 %}) of Homebrew, we are now choosing to advertise the `HOMEBREW_INSTALL_FROM_API` environment variable to the general public. A lot of work was done by @Rylan12 to make the experience more stable and the API install method is bound to bring a big performance improvement to those who dare enable it.
```

Comment:
```suggestion
Since version [3.6.0]({% post_url 2022-09-07-homebrew-3.6.0 %}) of Homebrew, we are now choosing to advertise the `HOMEBREW_INSTALL_FROM_API` environment variable to the general public. A lot of work was done by @Rylan12 to make the experience more stable and the API install method is bound to bring a big performance improvement to those who are willing to give it a try.
```

I don't think it matters much, but I worry that the word "dare" gives the impression that this is still in a very early beta with lots of bugs when in reality it's actually pretty close to being fully ready